src/Tribe/Customizer.php

```diff
@@ -485,7 +485,7 @@ public function inline_style( $force = false ) {
 				/**
 				 * Fires before a style is, possibly, printed inline depending on the stylesheet.
 				 *
-				 * @since TBD
+				 * @since 4.12.15
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 4d38aa0d80a394fdb2e81f96396460ce</sub>